dockerfile/cuda11.1.1.dockerfile

```diff
@@ -2,6 +2,7 @@ FROM nvcr.io/nvidia/pytorch:20.12-py3
 
 # OS:
 #   - Ubuntu: 20.04
+#   - OpenMPI: 4.0.5
```

Comment:
Where is the OpenMPI installation?

Reply:
It will be installed into system path: `/usr/local/bin/`

Reply:
I mean there is no code for OpenMPI installation.

Reply:
For nvidia image, it is installed in base image. Just list the version in comment.